frontend/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -226,6 +230,7 @@ export class DeckGlJsonChart extends PureComponent<PropsWithHeight, State> {
           width={deck.initialViewState.width}
           layers={this.state.initialized ? deck.layers : []}
           getTooltip={this.createTooltip}
+          ContextProvider={MapContext.Provider}
```

Comment:
I tend not to like importing private items (as demo with the preceding _). I did some research on this. I would love a comment in here that suggests:

1. Why this is necessary (supposedly related to the Navigation Control)?
2. Why we should not worry about the private method?

Looking at (2) Yarn.lock resolves to 5.2.12, and this https://github.com/visgl/react-map-gl/blob/v5.2.12/src/index.js#L54 suggests it's experimental. So I would take some precautions here:

e.g. perhaps pin the package because minor version bumps might break it

Reply:
Is a comment necessary still after importing the public `MapContext`?

Reply:
Nope